hazelcast/src/main/java/com/hazelcast/internal/serialization/impl/SerializationUtil.java

```diff
@@ -60,6 +61,9 @@ static RuntimeException handleException(Throwable e) {
         if (e instanceof HazelcastSerializationException) {
             throw (HazelcastSerializationException) e;
         }
+        if (e instanceof HazelcastInstanceNotActiveException) {
```

Comment:
this is a broader behaviour change - we do not want to wrap `HazelcastInstanceNotActiveException` inside `HazelcastSerializationException`